.github/workflows/dispatcher.yml

```diff
@@ -0,0 +1,107 @@
+name: dispatcher
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, ready_for_review, reopened]
+    
+permissions:
+  actions: write
+  pull-requests: read
+  contents: read
+
+jobs:
+  dispatcher:
+    runs-on: ubuntu-latest
+
+    outputs:
+      workflowToRunMap: ${{ steps.parse.outputs.result }}
+      pr_number: ${{ steps.set-pr-data.outputs.pr_number }}
+      pr_ref: ${{ steps.set-pr-data.outputs.pr_ref }}
+
+    steps:
+      - name: Extract PR metadata
+        id: set-pr-data
+        run: |
+          echo "pr_number=${{ github.event.pull_request.number }}" >> $GITHUB_OUTPUT
+          echo "pr_ref=${{ github.event.pull_request.head.ref }}" >> $GITHUB_OUTPUT
+
+      - name: Parse PR description and trigger workflows
+        id: parse
+        uses: actions/github-script@v8
+        with:
+          result-encoding: string
+          script: |
+            const pr = context.payload.pull_request;
+            const prBody = pr.body || "";
+
+            if (!prBody) {
+              core.info('PR body is empty. No test will be started.');
+              return JSON.stringify({});
+            }
+
+            if (prBody.includes('NO_TEST')) {
+              core.info('PR description contains `NO_TEST`. Skipping testing axes.');
+              return JSON.stringify({});
+            }
+
+            const workflowMap = {
+              'CORE': 'core.yml',
+              'AS_TESTS': 'as_tests.yml',
+              'RTS': 'rts.yml',
+              'JACOCO': 'jacoco.yml',
+              'XTS': 'xts.yml',
+              'QA_JTA': 'qa_jta.yml',
+              'QA_JTS_OPENJDKORB': 'qa_jts.yml',
```

Comment:
@jmfinelli what's the plan for these workflows?

Reply:
Sure, I will modify this PR